docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -293,6 +293,14 @@ repository. If you wish to search data across multiple regions, configure
 multiple clusters and use <<modules-cross-cluster-search,{ccs}>> or
 <<xpack-ccr,{ccr}>> instead of {search-snaps}.
 
+It's worth noting that if a searchable snapshot index has no replicas, then when the node
```

Comment:
I'd delete "It's worth noting that" and start the sentence as is without it -- you're writing it, so it's obviously worth noting :) 

Should this be a WARNING block as well, like directly above? Seems a little strange to have text -> warning -> text again; and this is sort of a warning.

Reply:
I merged just before your comment, apologies!

Reply:
No worries, looks like a near perfect race condition 😄